sosia/utils/logger.py

```diff
@@ -0,0 +1,20 @@
+"""Module to configure the logging for the package."""
+import logging
+from pathlib import Path
+
+_cache_folder = Path.home() / '.cache' / 'sosia/'
+_cache_folder.mkdir(parents=True, exist_ok=True)
+
+logger = logging.getLogger('sosia')
+
+file_handler = logging.FileHandler(_cache_folder / 'sosia.log', mode='a', encoding='utf-8')
```

Comment:
If this is append mode, will it always write to a new file? Ideally I want a fresh file when I initiate a new `Original()`. It's okay when it overwrites the old stuff, but it's not ok when the file grows forever.